### PR TITLE
2016 is change by create individual

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Project conversion comming.

File: src/PKSim.Core/Model/PKSimParameter.cs [hunk 1]
```diff
@@ -28,96 +27,9 @@ public override double Value
                IsFixedValue = false;
          }
       }
-
-      public override bool IsChangedByCreateIndividual
```
Comment (msevestre) [inline src/PKSim.Core/Model/PKSimParameter.cs:32]: Not needed anymore...only for project conversion

File: src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs [hunk 2]
```diff
@@ -152,5 +153,68 @@ private void addExpressionProfilesUsedBySimulationSubjectToProject(ISimulationSu
 
          _converted = true;
       }
+
+      private bool isChangedByCreatedIndividual(IParameter parameter)
```
Comment (msevestre) [inline src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs:157]: moved it here for now... I am going to use it do convert a project to v11.
ALternatively, we could read the info from the db but I do not know how I can retreive the ParameterInfo from DB for a given parmaeter instance? I am sure we have a way somewhere. Do you know @Yuri05 ?

**Comment by msevestre** [comment]
This allows to mak ethis

image


COOL! Dynamic parameters changes by the Diesease State

File: src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs [hunk 3]
```diff
@@ -105,14 +111,37 @@ private void addOriginDataNodeToOriginData(string value, string unit, string nod
...
+         if (individual == null)
+            return;
+
+         addEstimatedGFRParameterTo(individual);
+         updateIsChangedByCreatedIndividualFlag(individual);
+      }
+
+      private void updateIsChangedByCreatedIndividualFlag(Individual individual)
+      {
+         var defaultHuman = _defaultIndividualRetriever.DefaultHuman();
```
Comment (msevestre) [inline src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs:135]: @Yuri05 Just creating a standard invidual and updating from there. Easy

**Review by msevestre — COMMENTED** [review]
Merging this as soon as it builds

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**